src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -58,6 +58,8 @@ public class TestConnectionCommand : PSCmdlet, IDisposable
 
         private static byte[]? s_DefaultSendBuffer;
 
+        private CancellationTokenSource _dnsLookupCancel = new CancellationTokenSource();
```

Comment:
From Codacy:
```suggestion
        private readonly CancellationTokenSource _dnsLookupCancel = new CancellationTokenSource();
```

Reply:
Huh, I thought I... ah, I must have taken it out again at some point. Thanks!